kornia/filters/kernels.py

```diff
@@ -3,6 +3,8 @@
 
 import torch
 
+from kornia.constants import pi
```

Comment:
```suggestion
import math
```

Reply:
Why do we have pi in kornia then?

Reply:
in initial pytorch versions floats were not really promoted with tensors